ChangeLog

```diff
@@ -1,3 +1,6 @@
+Version 7.8.1
```

Comment:
I see, thanks; I think it could be confusing, though, if there are changes included in a version which were committed after the commit with the corresponding version bump.